doto-backend/index.js

```diff
@@ -1,68 +1,67 @@
-const express = require('express')
-const cors = require('cors')
-require('dotenv').config();
-const passportSetUp = require('./src/config/passport-setup.js')
-const app = express()
-const apiPort = process.env.PORT || 3001
-const passport = require('passport')
-const winston = require('winston');
-const expressWinston = require('express-winston');
-const { logger } = require('./src/common/logging');
+const express = require("express");
+const cors = require("cors");
+require("dotenv").config();
+require("./src/config/passport-setup.js");
+const app = express();
+const apiPort = process.env.PORT || 3001;
+const passport = require("passport");
+const winston = require("winston");
+const expressWinston = require("express-winston");
+const { logger } = require("./src/common/logging");
```

Comment:
Why double quotes instead of single quotes? My understanding was that these should be single quotes.

Reply:
Its a personal preference. The style guidelines setup by the previous group had double quotes, so we should stick to their guidelines.

Reply:
Agreed.. Single quotes is better suited in JS

Reply:
@PreetPatel yes I agree, thats how I set it up for the last project. However, as new contributors into this project, we shouldn't change style guidelines based on our personal preferences.

Reply:
But if we're adding linting where it didn't exist previously, I think that allows us the freedom to define new style guidelines.

Reply:
They have already defined the style for the frontend package. I have taken the same style guidelines for the backend package. I don't see the value in modifying the style for one of the two packages, which would be inconsistent. Moreover, we shouldn't change the style for both packages because that would be changing their set style guidelines.